internal/verify/test_fixture.go

```diff
@@ -82,15 +85,18 @@ func loadResources(fsys fs.FS, path string) (*Resources, error) {
 		return nil, err
 	}
 
+	resources := &Resources{
+		FilePath: fp,
+	}
+
 	pb := &policyv1.TestFixture_Resources{}
 	if err := loadFixtureElement(fsys, fp, pb); err != nil {
-		return nil, err
+		resources.LoadError = err
```

Comment:
I'd probably accumulate the errors as follows:

```go
resources.LoadError = multierr.Append(resources.LoadError, err)
```

Reply:
I did that and then removed it because it's not used down the chain at the mo. #IndecisiveSam 😫

Reply:
No worries. It can always be added later if needed.